docs/fsharp/get-started/get-started-vscode.md

```diff
@@ -42,58 +42,50 @@ You must also install the [.NET Core SDK](https://www.microsoft.com/net/download
 
```

Comment:
should you change ms.date?

not related to your change: line 8 - change order and typos
IDE (Integrade Development Enivronment) -> Integrated Development Environment (IDE)

I'd also run Acrolinx on this article to make sure there are no further updates needed. A lot of my suggestions would be caught by that.

Reply:
Changed ms.date and this suggestion. I can run acrolinx today as well.